usaspending_api/etl/tests/integration/test_load_transactions_in_delta.py

```diff
@@ -364,12 +352,35 @@ def verify(
                     TestInitialRun.verify_raw_vs_int_tables(spark, table_name, col_names)
 
     @mark.django_db(transaction=True)
-    def test_edge_cases_using_only_source_tables(
-        self, spark, s3_unittest_data_bucket, hive_unittest_metastore_db, _populate_initial_source_tables_pg
-    ):
-        # 1. Call initial run with no raw tables, except for published_fabs and detached_award_procurement.
-        # Also, don't do initial copy of tables
-        TestInitialRun.initial_run(s3_unittest_data_bucket, initial_copy=False)
+    def test_edge_cases_using_only_source_tables(self, spark, s3_unittest_data_bucket, hive_unittest_metastore_db):
+        # Setup some source tables without data, this test does not require these tables to be populated
+        raw_db = "raw"
+        spark.sql(f"create database if not exists {raw_db};")
+        spark.sql(f"use {raw_db};")
+        spark.sql(
+            TABLE_SPEC["published_fabs"]["delta_table_create_sql"].format(
+                DESTINATION_TABLE="published_fabs",
+                DESTINATION_DATABASE=raw_db,
+                SPARK_S3_BUCKET=s3_unittest_data_bucket,
+                DELTA_LAKE_S3_PATH=CONFIG.DELTA_LAKE_S3_PATH,
+            )
+        )
+        spark.sql(
+            TABLE_SPEC["detached_award_procurement"]["delta_table_create_sql"].format(
+                DESTINATION_TABLE="detached_award_procurement",
+                DESTINATION_DATABASE=raw_db,
+                SPARK_S3_BUCKET=s3_unittest_data_bucket,
+                DELTA_LAKE_S3_PATH=CONFIG.DELTA_LAKE_S3_PATH,
+            )
+        )
```

Comment:
More of stylistic comment, but would it be cleaner to create a list of table names and then just loop over them instead of have all this code for each table that you are creating? something like `for table in tables` then just use `table` for the TABLE_SPEC and DESTINATION_TABLE values